frontend/src/components/SectionLine/SectionLine.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types'
+
+import LineChart from 'components/LineChart'
+import { LayoutMetricType } from 'utils/propTypes'
+
+const SectionLine = ({ layout, data }) => {
+  const xAxisKey = layout.x?.key
+  const yAxisKey = layout.y?.key
+  const chartData = data.map(item => ({
+    x: item[xAxisKey],
+    y: item[yAxisKey],
+  }))
+
+  return <LineChart data={chartData} title={layout.title} xAxisLabel="Population" yAxisLabel="Year" />
```

Comment:
Could you alter detail_layout_example_small_business_support.json to include `title` values for the axis labels?
```json
{
  "title": "Projected Population",
  "key": "pop_proj",
  "type": "line",
  "x": {
    "key": "year",
    "title": "Year"
  },
  "y": {
    "key": "pop",
    "format": "number",
    "title": "Population"
  }
}
```